recipe/meta.yaml

```diff
@@ -1,29 +1,42 @@
 {% set name = "plumbum" %}
-{% set version = "1.8.1" %}
+{% set version = "1.9.0" %}
+{% set tests_to_skip = "" %}
+{% set tests_to_skip = tests_to_skip + "test_cwd or test_mixing_chdir or test_contains or 
+                                        test_path or test_iter_lines_buffer_size" %}
+{% set tests_to_skip = tests_to_skip + " or test_runfile_rich" %}       # [linux]
+{% set tests_to_skip = tests_to_skip + " or test_help_lang" %}          # [osx]
+{% set tests_to_skip = tests_to_skip + " or test_nolang_switches" %}    # [osx]
+{% set tests_to_skip = tests_to_skip + " or test_failure" %}            # [osx]
+{% set tests_to_skip = tests_to_skip + " or test_home" %}               # [win]
+
+
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: 88a40fc69247d0cd585e21ca169b3820f46c484535102e16455d2202727bb37b
+  sha256: e640062b72642c3873bd5bdc3effed75ba4d3c70ef6b6a7b907357a84d909219
 
 build:
-  noarch: python
+  skip: True                      # [py<38]
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
     - hatchling
     - hatch-vcs
   run:
-    - python >=3.6
-    - pywin32-on-windows
-    - paramiko
+    - python
+    - pywin32                     # [win]
+    - importlib_resources         # [py<39]
+  run_constrained:
+    - sphinx >=4.0.0
+    - sphinx-rtd-theme >=1.0.0
```

Comment:
```suggestion
```
These are for building the upstream docs and are not relevant to the library itself.